### PR TITLE
Add foreman_openscap to list of redmine issue required

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by domcleal** [comment]
@shlomizadok you're the plugin maintainer, did you want this?  Not all commits and PRs appear to do it today.  See http://projects.theforeman.org/projects/foreman/wiki/PrProcessor#Commit-message-styles for details.

**Comment by ehelms** [comment]
I should have mentioned I based this update off of https://github.com/theforeman/foreman_openscap/pull/154#issuecomment-196313624  assuming that the "reaction" was agreement. Can't say I am a fan of those reactions at the moment.

**Comment by shlomizadok** [comment]
I'm too low at the food chain. Up for you to decide.

**Comment by domcleal** [comment]
As far as I'm concerned, it's up to the plugin maintainer (you) - there's no requirement for Foreman plugins to use Redmine (mine don't), different plugins are developed in different ways.  If you want to enforce that contributors file tickets for every commit, I'm happy to merge this for you.

**Comment by shlomizadok** [comment]
Never signed-up to be that plugin maintainer. Hopefully a successful one will become soon.
As far I know, foreman_openscap does not have issues enabled so it makes sense to enforce Redmine issues. Yet again, totally up to you.

**Comment by ares** [comment]
@shlomizadok @domcleal I'd appreciate this being merged. If foreman_openscap does not have official maintainer, for me as an occasional contributor it would make my life easier, I have to link PRs manually, Issues that are solved are not being closed etc.

File: config/redmine_issue_required_repos.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@
 - theforeman/foreman-docker
 - theforeman/foreman-selinux
 - theforeman/foreman-tasks
+- theforeman/foreman_openscap
 - theforeman/smart_proxy_dynflow
 - theforeman/smart_proxy_remote_execution_ssh
```
Comment (ares) [inline config/redmine_issue_required_repos.yaml]: could you please add `theforeman/smart_proxy_openscap` as well?

**Comment by domcleal** [comment]
@ares those are different mechanisms to this. To link PRs to tickets, the webhook is needed (http://projects.theforeman.org/projects/foreman/wiki/Jenkins#PR-scanner-repo-hooks) and to close tickets, the Git repository must be configured in the Redmine project settings.

This PR is proposing making Redmine tickets mandatory and enforcing commit message styles.

**Comment by ares** [comment]
I see, I already set repositories in redmine so revisions are created but issues does not seem to be closed even though commit included Fixes #nnn. This might not be that important, but I guess every commit should have its own issue anyway.

Reasons:
* consistency with other repositories
* documentation of changes
* we encourage people to do so already https://github.com/theforeman/smart_proxy_openscap/pull/27#issuecomment-207411671

**Comment by shlomizadok** [comment]
@domcleal - please add merge this to enforce commit message styles. Thanks

**Comment by domcleal** [comment]
The PR needs to be rebased, and @ares' comment about the smart proxy plugin addressed.

**Comment by ares** [comment]
I rebased and added the smart_proxy_openscap at #40, please close this one. I believe @ehelms would not mind :-)

**Comment by ehelms** [comment]
Closing in favor of #40 -- thanks @ares ! Sorry for forgetting about this.

**Comment by ares** [comment]
no worries :-)

**Pull request closed without merging**